test/functional/feature_longterm_lockin.py

```diff
@@ -123,6 +123,9 @@ def run_test(self):
         result5 = self.nodes[0].getmasternode(nodeid5)
         assert_equal(result5[nodeid5]['state'], 'ENABLED')
 
+        # Timelock should no longer be present
+        assert_equal('timelock' not in result5[nodeid5], True)
+
         # Resign 5 year MN
```

Comment:
Can you add a test case that upon expiry of timelock, if the masternode does not resign, the multiplier does not accrue at faster speed. 

1. Get accrual speed before expiry. Ensure it's 1.5x or 2x.
2. Set mock time to past expiry.
3. Get accrual speed. It should still be 1.5x or 2x.
4. Publish a block from the masternode.
5. Get accrual speed. It should still now be 1x

Reply:
That test case is a problem in a Python test script, we cannot stake using the MNs we create in those scripts. We can see that the timelock is missing so that once the multiplier is wiped it should not accrue anymore, but starting a node using a masternode created in a test case is problematic.